core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -432,7 +442,10 @@ protected void configurePipeline(ChannelHandlerContext ctx, String protocol) thr
         private void addHttp2Handlers(ChannelHandlerContext ctx) {
             final ChannelPipeline p = ctx.pipeline();
             p.addLast(newHttp2ConnectionHandler(p, SCHEME_HTTPS));
-            p.addLast(new HttpServerHandler(config, gracefulShutdownSupport, null, H2, proxiedAddresses));
+            final HttpServerHandler httpServerHandler = new HttpServerHandler(configHolder,
+                    gracefulShutdownSupport,
+                    null, H2, proxiedAddresses);
```

Comment:
nit: I think the code style seems not to abide by our code formatter.
It should be either
```suggestion
            final HttpServerHandler httpServerHandler = new HttpServerHandler(configHolder,
                                                                              gracefulShutdownSupport,
                                                                              null, H2, proxiedAddresses);
```
or
```suggestion
            final HttpServerHandler httpServerHandler = new HttpServerHandler(
                    configHolder, gracefulShutdownSupport, null, H2, proxiedAddresses);
```